### PR TITLE
Update minimal-apis.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

File: aspnetcore/fundamentals/minimal-apis.md [hunk 1]
```diff
@@ -1598,6 +1598,8 @@ app.MapGet("/old-path", () => Results.Redirect("/new-path"));
 app.MapGet("/download", () => Results.File("myfile.text"));
 ```
 
+<a name="binr7"></a>
```
Comment (brunolins16) [inline aspnetcore/fundamentals/minimal-apis.md:1601]: @Rick-Anderson  By any chance this change is to fix the _anchor_ link? `#built-in-results-1`

Comment (Rick-Anderson) [reply aspnetcore/fundamentals/minimal-apis.md:1601]: Yes, guarantees it will work.

Comment (Rick-Anderson) [reply aspnetcore/fundamentals/minimal-apis.md:1601]: When .NET 7 is released I'll move the .NET 7 version to the top and that will change the anchor.

Comment (Rick-Anderson) [reply aspnetcore/fundamentals/minimal-apis.md:1601]: I'm almost finished with https://github.com/dotnet/AspNetCore.Docs/pull/27157/files. Really nice work.

Comment (Rick-Anderson) [reply aspnetcore/fundamentals/minimal-apis.md:1601]: Don't change your PR, I submitted a suggestion to fix it.

Comment (brunolins16) [reply aspnetcore/fundamentals/minimal-apis.md:1601]: Thanks!!!

**Pull request merged**